.travis.yml

```diff
@@ -28,6 +28,10 @@ jobs:
 # install:
 #  - "cd fe; npm install; cd .."
 
+branches:
+  only:
+    - main
```

Comment:
You forgot this

> 2. feature branches only when they have an open pr

Reply:
Actually it is still running on this branch? So does it work? Idk, I'm kinda confused. Please make sure it will work on **feature** branches when they have a pr

Reply:
Pull requests are seperate of the branches specified

Reply:
Ah okay, good to know. Then merge away!